src/corg/working-on-nuxeo-sources/unit-testing.md

```diff
@@ -204,22 +204,20 @@ history:
 ---
 ## Overview
 
-The Nuxeo Test Framework is based on JUnit4 and provides features like:
+The Nuxeo Test Framework is based on JUnit4 and provides features such as:
 
 *   Test configuration through annotations,
 *   Embedded Nuxeo Runtime,
 *   [WebDriver](http://docs.seleniumhq.org/projects/webdriver/) support (and optional [Concordion](http://www.concordion.org/) support),
 *   [Guice](http://code.google.com/p/google-guice/) injection of Nuxeo services and other configuration objects (injection is available only in test classes),
 *   High extensibility and test configuration re-usability.
 
-At the time of this writing, the test framework depends on JUnit 4.7, Guice 1.0 and WebDriver 0.9.7376.
+At the time of this writing, the test framework depends on JUnit 4.12, Guice 3.0 and WebDriver 0.6.
```

Comment:
We didn't downgrade WebDriver :)
The reference was `org.seleniumhq.webdriver` artifacts.
The new corresponding artifacts are `org.seleniumhq.selenium` artifacts. See https://jira.nuxeo.com/browse/NXP-8549
The new version is `2.53.0`.
Worth to mention the `org.seleniumhq.selenium:htmlunit-driver:2.20` which versioning forked from the other libraries (see https://www.versioneye.com/java/org.seleniumhq.selenium:htmlunit-driver/2.27).


```
$ git show f0ce2d4d:pom.xml|grep -B2 webdriver.version
  <properties>
    <webdriver.version>0.9.7376</webdriver.version>
--
        <groupId>org.seleniumhq.webdriver</groupId>
        <artifactId>webdriver-htmlunit</artifactId>
        <version>${webdriver.version}</version>
--
        <groupId>org.seleniumhq.webdriver</groupId>
        <artifactId>webdriver-ie</artifactId>
        <version>${webdriver.version}</version>
--
        <groupId>org.seleniumhq.webdriver</groupId>
        <artifactId>webdriver-firefox</artifactId>
        <version>${webdriver.version}</version>
--
        <groupId>org.seleniumhq.webdriver</groupId>
        <artifactId>webdriver-chrome</artifactId>
        <version>${webdriver.version}</version>
--
        <groupId>org.seleniumhq.webdriver</groupId>
        <artifactId>webdriver-support</artifactId>
        <version>${webdriver.version}</version>
--
        <groupId>org.seleniumhq.webdriver</groupId>
        <artifactId>webdriver-common</artifactId>
        <version>${webdriver.version}</version>
```

Reply:
I got this info from another dev as we looked at the pom together... Not sure what you would like me to replace this with. WebDriver 2.53.0? Selenium 2.53.0 ?

Reply:
the other dev mislooked :) yes, something like this